website/app/styles/pages/application/tabs.scss

```diff
@@ -36,11 +36,15 @@
   @include breakpoint.x-large() {
     padding: 24px var(--doc-page-stage-gutter-x-large) 0;
   }
+}
 
-  // Make tabs position sticky/fixed if the viewport height is greater than 480px
+// Make tabs position sticky/fixed if the viewport height is greater than 480px and dialog is NOT open
+.doc-page-wrapper:not(:has(dialog[open])) {
   @media (height >= 480px) {
-    position: sticky;
-    top: var(--doc-page-header-height);
+    .doc-page-tabs {
+      position: sticky;
+      top: var(--doc-page-header-height);
+    }
```

Comment:
I tried every combination of z-indexes on various elements including the code blocks and the only solution I could come up with was this. (To just unstick tabs when a dialog is open.)

Reply:
I think that might be our only option in the current layout. While not ideal, it does solve the problem for the end user.